docs/about/introduction.adoc

```diff
@@ -16,102 +16,120 @@
 
 ///////////////////////////////////////////////////////////////////////////////
 
-= About Helidon
-:description: about Helidon
+= Introduction
+:description: Helidon
 :keywords: helidon, java, microservices, microprofile
 :rootdir: {docdir}/..
 
 include::{rootdir}/includes/attributes.adoc[]
 
-Helidon is a collection of Java libraries for writing microservices that run on a fast web core powered by Netty. Its available in two frameworks: Helidon SE and Helidon MP.
+== Contents
 
-== A Collection of Java Libraries
+* <<What is Helidon?, What is Helidon?>>
+* <<Helidon Flavors, Helidon Flavors>>
```

Comment:
In 2.x documentation we never used "flavor". We used "programming model" which I prefer.

Reply:
We use _flavor_ in all presentations. Upcoming Nima will have the same programming model as MP (blocking with annotations). Using _programming model_  as differenciator will not make sense.